core/start-companion-core

```diff
@@ -7,6 +7,7 @@ SERVICES=(
     'cable_guy',"$SERVICES_PATH/cable_guy/main.py"
     'wifi',"$SERVICES_PATH/wifi/main.py"
     'autopilot',"$SERVICES_PATH/ardupilot_manager/main.py"
+    'autopilot-rest_api',"$SERVICES_PATH/ardupilot_manager/rest_api.py"
```

Comment:
How autopilot and autopilot-rest-_api are going to run concurrently ?

Reply:
Humnnn, this is a flaw. In this implementation, they are not.

I think the best way to do it is to [make the singleton implementation thread-safe](https://stackoverflow.com/questions/50566934/why-is-this-singleton-implementation-not-thread-safe).

Reply:
Can't you just remove the main.py line or replace main.py with rest_api ?
I'm not sure if I follow what is the problem here

Reply:
Oh, ok, now I got it. It makes much more sense, for sure.